cmake/thirdParties.cmake

```diff
@@ -0,0 +1,19 @@
+find_package(Threads)
```

Comment:
What is Threads needed for?

Reply:
I don't think it is needed. It was in the original PR, but I removed it now.